main-app/app/scripts/services/game-api.js

```diff
@@ -36,13 +36,13 @@
 
                 var deferred = $q.defer();
                 var serverCallInformation = new ServerCallInformation('http://EUTAVEG-01.tombola.emea:35000/api/v1.0/makemove', {
-                    playerNumber: playerNumber,
+                    playerNumber: gameModel.currentPlayer,
```

Comment:
Don't pass the gameModel here - it increases coupling (i.e. the gameApi service depends on you sending an object through which has a "currentPlayer" member) - just pass through the current player number directly as a number.